src/libcore/syncAccount.js

```diff
@@ -72,8 +71,7 @@ export async function syncCoreAccount({
 
     return account;
   } catch (e) {
-    if (e.name !== "Error") throw remapLibcoreErrors(e);
-    throw new SyncError(e.message);
+    throw remapLibcoreErrors(e);
```

Comment:
I would recommend this:

```js
const betterError = remapLibcoreErrors(e);
if (betterError.name !== "Error") throw betterError; // we found a better error to remap to
throw new SyncError(betterError.message); // otherwise we fallback into a SyncError
```

i think the initial idea (that was later broken) is that if we can't find a "precise" error (which maps to a wording, like a no network case) we will fallback into a "SyncError" (that have a "good" wording for the user)